tests/stdlib/tstrutil.nim

```diff
@@ -187,6 +187,20 @@ proc testRFind =
   assert "0123456789ABCDEFGAH".rfind({'0'..'9'}, start=5) == 9
   assert "0123456789ABCDEFGAH".rfind({'0'..'9'}, start=10) == -1
 
+proc testTrimZeros() =
+  var x = "1200"
+  x.trimZeros()
+  assert x == "1200"
+  x = "120.0"
+  x.trimZeros()
+  assert x == "120"
+  x = "0."
+  x.trimZeros()
+  assert x == "0"
+  x = "1.0e2"
+  x.trimZeros()
+  assert x == "1e2"
```

Comment:
All these tests are either integers or floats of `x.0` format.

I would like to see more tests with different floats, just to be sure there's no some subtle bug hiding somewhere. Some floats I'd like to see tested (if you want, add more):

```
78.90
1.23e4
1.01
1.1001
0.0
0.01
1e0
```

Reply:
I would add to the list ``70.85`` but I forgot